CoreUtilities/src/au/gov/asd/tac/constellation/utilities/json/JsonUtilities.java

```diff
@@ -302,9 +334,11 @@ public static String getTextValues(String attribute, JsonNode node, String delim
      * @return a {@code String} or null if not found
      */
     public static String getTextValueOfFirstSubElement(String attribute, String innerAttribute, JsonNode node) {
-        if (node.has(attribute)) {
+    
+        if (node.has(attribute)) {          
             if (node.get(attribute).has(0) && node.get(attribute).get(0).has(innerAttribute)) {
-                return node.get(attribute).get(0).get(innerAttribute).textValue();
+                JsonNode innerNode = node.get(attribute).get(0).get(innerAttribute);
```

Comment:
final

Reply:
just removed the variable entirely and rolled it into return statement